composer.json

```diff
@@ -8,12 +8,16 @@
     "require": {
         "cweagans/composer-patches": "^1.6",
         "drupal/address": "^1.8",
+        "drupal/crop": "^2.1",
         "drupal/entity_browser": "^2.5",
         "drupal/entity_usage": "^2.0@beta",
+        "drupal/field_formatter_class": "^1.4",
         "drupal/field_group": "^3.0",
+        "drupal/fontawesome": "2.x-dev#0a02cad1",
```

Comment:
Is there a reason for pinning Fontawesome to a particular commit?

Reply:
> For this I have created a feature/homepage branch

+1

> Is there a reason for pinning Fontawesome to a particular commit?

I am being lazy and copy-pasting from the [commit message](https://github.com/localgovdrupal/localgov_paragraphs/pull/6/commits/46d9981ba2262076536b758236b7300c304e1024):

> The 2.x-dev#0a02cad1 version of fontawesome contains necessary schema definitions for our tests to pass.  We should revert to 'drupal/fontawesome:^2.17' as soon as drupal/fontawesome:2.18 is out.